### DISCUSSION TITLE
GameCube does not power ESP32 correctly

### DISCUSSION THREAD

**Discussion by bullseye33** [darthcloud/BlueRetro#253] [Q&A]
Hello everyone, I built the BlueRetro device for GameCube using an ESP32-DevKitC-32E and while powering everything through micro usb I can get it working (I do only plug data line to the GC on pin 19), state led blinks correctly while pairing etc.. Then if I try to get power from the GC it does not work and the state led blinks very slowly (like once every 8-10 seconds). I measured output voltage from the controller cord that I'm using and I get 5V when I don't plug it to the esp module, 4.6-4.8V when it's plugged to the esp module. I'm afraid that it can't get enough power to boot up, can someone help me figure out why?
If it's relevant to diagnose the problem I'm using a Japanese GC with Xeno Mod, GB Player plugged and the power supply I'm using is a cheap Amazon one since I can't use the original 110V psu in Europe.
Thanks in advance!

**Comment by darthcloud** [comment]
Unlikely, I think you just need to stabilize the unused pins (5, 26, 27) as indicated in assembly instructions:
https://github.com/darthcloud/BlueRetro/wiki/BlueRetro-Cables-Build-Instructions#assembly-instructions-30

**Reply by bullseye33** [reply]
You were right, I completely missed that assembly part. I soldered pin 5, 26 and 27 to 3.3v of the controller cable and still nothing. I used BlueRetro_gc_spiffs.bin firmware so I guess I don't have to ground all the other IO pins... any other idea? Thanks for your answer btw

**Reply by darthcloud** [reply]
Post some pictures were I can see the connection easily.

**Reply by bullseye33** [reply]
![IMG_20220417_234424__01](https://user-images.githubusercontent.com/50544102/163733254-2616c81d-745e-4f5b-82de-70be3aae823b.jpg)
This is the contraption that I made to test everything. Sorry if cables are not color coded but I ran out of colors 😓

**Reply by darthcloud** [reply]
Don't use GC 3.3v leave unconnected, use the 3.3v from the ESP32 at the top.

**Reply by bullseye33** [reply]
Okay, I'll try that and I'll let you know the result. Thank you

**Reply by bullseye33** [reply]
Update: soldered 3.3V from the ESP to the unused controller pins instead of using controller's 3.3V line and still the same issue
Here is a video https://youtu.be/22eegGeXmYc

**Comment by khtp** [comment]
I am using a wii the first model compatible with gamecube controllers and connect the data cable to IO19 the GND of the connector to GND of the esp32 and 5v to the VIN pin of the esp32 and then bridge pin D5,IO26,IO27 to 3.3v and that's it It works without problem and without using the micro usb to give it power, my wii is in the USA region

**Comment by khtp** [comment]
![278460577_529751125541059_3768465949262314114_n](https://user-images.githubusercontent.com/50355226/163733495-812976d4-6c25-413b-9171-405c9c97b30e.jpg)
 
this is how i have it for gamecube where i am currently using it on the wii

**Reply by bullseye33** [reply]
Cool, I also plan to 3d print a case but I'll remove gpio pins to have a slim design.
